telethon/events/messagedeleted.py

```diff
@@ -49,7 +51,9 @@ def build(cls, update, others=None, self_id=None):
             )
 
     class Event(EventCommon):
-        def __init__(self, deleted_ids, peer):
+        def __init__(self,
+                     deleted_ids: Optional[Sequence],
```

Comment:
Does `Sequence`, without a type, work? Anyway the IDs should be integers, so there's no reason not to do this.

```suggestion
                     deleted_ids: Optional[Sequence[int]],
```

Reply:
I just wasn't sure if there is some automatic casting from string or any other type to integer. You're right if only integers should be passed here